ApiBundle/Transformer/AreaTransformer.php

```diff
@@ -94,35 +100,42 @@ public function transform($area, NodeInterface $node = null, $parentAreaId = nul
                 'id' => $area->getAreaId()
             )
         );
+
         $facade->addLink('_self_form', $this->generateRoute('open_orchestra_backoffice_area_form', array(
             'nodeId' => $node->getId(),
             'areaId' => $area->getAreaId(),
         )));
-        $facade->addLink('_self_block', $this->generateRoute('open_orchestra_api_area_update_block', array(
-            'nodeId' => $node->getId(),
-            'areaId' => $area->getAreaId()
-        )));
+
+        if ($this->authorizationChecker->isGranted(TreeNodesPanelStrategy::ROLE_ACCESS_UPDATE_NODE, $node)) {
```

Comment:
on second thougt, at this point, I think that you should use the `AuthorizeEditionManager` instead of the `AuthorizationChecker`.

You are checking a right for an edition and not only for a right, this way the authorizeEditionManager has been built for that